### PR TITLE
Add Indie Thin stack

### PR DESCRIPTION
**Pull request by kory-smith**

### CODE REVIEW

File: content/stacks.json [hunk 1]
```diff
@@ -95,6 +95,12 @@
       "description": "The Remix Stack for GraphQL developers, brought by Vulcan.",
       "tags": ["vercel", "graphql", "storybook", "mongo", "codesandbox"],
       "url": "https://github.com/VulcanJS/eurodance-stack"
+    },
+    {
+      "name": "Indie Thin Stack",
+      "description": "The Remix Stack for deploying to Fly or simple Node server with authentication, testing, linting, formatting, etc. Great for integrating with headless CMS",
+      "tags": ["fly", "docker", "github actions", "tailwind", "cypress", "msw", "vitest", "testing library", "prettier", "eslint", "typescript"],
+      "url": "https://github.com/VulcanJS/eurodance-stack"
```
Comment (nshki) [inline content/stacks.json]: This is linking to the Eurodance Stack, which I assume is incorrect?

Comment (kory-smith) [reply content/stacks.json]: Definitely incorrect. I just copy-pasted and forgot the URL on those two 🥲. Fixed in 2fc5b41.

**Comment by nshki** [comment]
Awesome, thanks for fixing that up! Would you mind fixing the merge conflict on this PR? I'm not able to do it since I need access to your repo, I believe.

**Comment by kory-smith** [comment]
6 months later and I did it :)

**Pull request merged**